src/main/java/com/management/HumanResources/dao/FirebaseDao.java

```diff
@@ -16,42 +16,38 @@
   private void init() {
     webClient =
         WebClient.builder()
-            .baseUrl("TBD")
-            .defaultHeaders(
-                httpHeaders ->
-                    httpHeaders.setBasicAuth(
-                        "TODO databaseUsername", "TODO databasePassword"))
+            .baseUrl("https://humanocity-{key}.firebaseio.com")
```

Comment:
I know you talked about it today and already wrote the instructions on Discord, but someone will mess this up and commit the key. I do think that it would be better to use an environment variable in this case if it is not too difficult to add in Java. We might use environment variables for other things later too.

Reply:
Haha I am fully in favor of having everyone setup with environment variables, but I don't know how difficult it would be for everyone to setup. But yea I can easily set this up to accept environment variables.

Reply:
I mean ideally there should be a file named `application.properties` I believe that should have one property per line or so in a `key=value` form. You could have a template file that includes the line `firebase_key={INSERT_KEY_HERE}`, commit it, then add the file to `.gitignore`, and just tell everyone to paste the key after they pull the commit to their machine.